ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,287 @@
+buildah_image() {
```

Comment:
Probably good to add a shebang here (#!/bin/sh) and possibly a line of comment or two.  Perhaps the name of the script and a quick line or two of it's expectations.